static/js/snap-pages/pages/AccountSnaps/AccountSnaps.tsx

```diff
@@ -0,0 +1,54 @@
+import { Notification, Strip } from "@canonical/react-components";
+import { RegisteredSnaps } from "../../components/RegisteredSnaps";
+import { PublishedSnapSection } from "../../components/PublishedSnapSection";
+import { useFetchAccountSnaps } from "../../hooks";
+
+export const AccountSnaps = () => {
```

Comment:
I think this should be a named function and the export at the bottom of the file to match other React components